src/Evolution/Systems/CurvedScalarWave/Worldtube/PunctureField.cpp

```diff
@@ -440,4 +443,1109 @@ void puncture_field_0(
+       1.0 / 2.0 * d_31 * d_37 * dv_164 * dv_180 * dv_183 *
+           (-d_10 * dv_182 - d_16 * dv_56 + dv_156 * r0 + dv_181) -
+       d_6 * dv_145 * dv_157 - d_81 * dv_136 * dv_178 - dv_136 * dv_145 -
+       dv_175 * (-d_3 * d_51 * dv_49 + d_3 * dv_69 - d_49 * dv_5 + d_49 * dv_7 +
+                 dv_58 * (d_45 * dv_171 + dv_172)));
+  get<0>(get<::Tags::deriv<CurvedScalarWave::Tags::Psi, tmpl::size_t<3>,
+                           Frame::Inertial>>(*result)) =
+      (1.0 / 6.0) * M * d_16 * d_82 * dv_184 * dv_216 * dv_280 +
+      (1.0 / 2.0) * M * d_26 * d_31 * d_37 * dv_164 * dv_180 * dv_293 * dv_72 *
+          dv_89 * r0 -
+      dv_175 * (d_125 * dv_58 + dv_290) - dv_280 * dv_281 - dv_280 * dv_291 -
+      dv_289 * (2 * d_125 * d_22 * d_41 * dv_65 +
+                4.0 * d_22 * d_41 * dv_283 * dv_58 - dv_288) -
+      dv_328 *
+          (d_10 * dv_226 * dv_295 + d_27 * dv_301 * dv_303 -
+           dv_198 * dv_236 * dv_243 * dv_312 - dv_295 * dv_296 +
+           dv_297 * dv_299 - dv_300 * dv_311 - dv_301 * dv_302 +
+           dv_304 * dv_306 + dv_304 * dv_313 + dv_307 * dv_308 -
+           dv_307 * dv_315 - dv_309 * dv_310 - dv_309 * dv_314 +
+           dv_327 *
+               (dv_199 *
+                    (d_109 *
+                         (d_128 * dv_319 +
+                          d_14 * (Dx * d_72 * (-dv_116 + dv_15 - dv_261) +
+                                  Dx * d_73 * dv_321 +
+                                  30 * Dy * d_2 * d_70 * dv_0 -
+                                  d_4 * dv_6 * (22 * dv_0 - dv_121 + dv_122) -
+                                  d_70 * dv_11 * dv_257 -
+                                  30 * d_71 * dv_0 * dv_54 - d_71 * dv_255) +
+                          dv_106 * (-4 * dv_269 + 7.0 * dv_270 + 3.0 * dv_6) +
+                          dv_294 * dv_320) +
+                     d_126 * dv_322 + dv_317 * dv_318 +
+                     dv_326 * (4 * d_1 * d_10 * d_13 * d_40 * dv_126 * dv_31 -
+                               d_1 * dv_324 - d_1 * dv_325 - d_128 * dv_323 -
+                               d_2 * dv_132 - d_2 * dv_262 +
+                               2.0 * d_23 * d_40 * d_46 * dv_126 * dv_286 -
+                               dv_263 * dv_317 -
+                               dv_265 * (-3 * dv_269 + 4.0 * dv_270 + dv_6))) +
+                dv_212 * dv_309 - dv_312 * dv_316));
+  get<1>(get<::Tags::deriv<CurvedScalarWave::Tags::Psi, tmpl::size_t<3>,
+                           Frame::Inertial>>(*result)) =
+      (1.0 / 6.0) * M * d_16 * d_82 * dv_184 * dv_216 * dv_333 +
+      (1.0 / 2.0) * M * d_26 * d_31 * d_37 * dv_164 * dv_180 * dv_341 * dv_72 *
+          dv_89 * r0 -
+      dv_175 * (-d_129 * dv_58 + dv_339) - dv_281 * dv_333 -
+      dv_289 * (-d_129 * d_48 * dv_66 + 4.0 * d_22 * d_41 * dv_276 * dv_58 -
+                dv_287 * dv_336 - dv_337) -
+      dv_291 * dv_333 -
+      dv_328 *
+          (12 * M * d_107 * d_27 * d_34 * dv_194 * dv_195 * dv_198 * dv_200 *
```

Comment:
Unclear if it's worth it, but here it looks like you could reduce the math a bit with some factoring of:
```
12 * M * d_107 * d_27 * d_34 * dv_194 * dv_200 * r0 *(...)
```
Just let me know if you don't care and then I'll just merge as-is :)

Reply:
oh wow, well spotted. weird that the sympy CSE did not pick that up. I squashed it in.